BookingsApi/BookingsApi/Controllers/V1/HearingsController.cs

```diff
@@ -209,6 +209,19 @@ public async Task<IActionResult> UpdateHearingsInGroup(Guid groupId, [FromBody]
                 
                 await _updateHearingService.UpdateEndpointsV1(requestHearing.Endpoints, hearing);
             }
+            
+            var hearings = request.Hearings.ToList();
+            var totalDays = hearings.Count;
+            var nextFirstHearingId = hearings[0].HearingId;
+
+
+            var nextFirstHearing = await _bookingService.GetHearingById(nextFirstHearingId);
+
+            if (nextFirstHearing == null)
```

Comment:
why would it be null? I don't think this is needed

Reply:
it is safe to check just in case it is null

Reply:
it's redundant. It will already be in the database as we are checking and updating it above

Reply:
removed